multi-select-list-item.js

```diff
@@ -235,10 +235,14 @@ class MultiSelectListItem extends RtlMixin(Localizer(LitElement)) {
 					<div class="d2l-labs-multi-select-list-item-text" aria-hidden="true">${this._getVisibleText(this.text, this.shortText, this.maxChars)}</div>
 					<d2l-offscreen>${this._getScreenReaderText(this.text, this.shortText)}</d2l-offscreen>
 				</div>
-				<d2l-icon role="button" aria-label="${this.localize('picker_remove_value', 'value', this.text)}" class="d2l-labs-multi-select-delete-icon" icon="d2l-tier1:close-large-thick" ?hidden="${!this.deletable}" @click="${this._onDeleteItem}"></d2l-icon>
+				<d2l-icon role="button" aria-hidden="true" class="d2l-labs-multi-select-delete-icon" icon="d2l-tier1:close-large-thick" ?hidden="${!this.deletable}" @click="${this._onDeleteItem}"></d2l-icon>
```

Comment:
It is technically improper to have both `aria-hidden` and `hidden` set to true. So I could do this, but I think it's more confusing to read.
```suggestion
				<d2l-icon role="button" ?aria-hidden="${this.deletable}" class="d2l-labs-multi-select-delete-icon" icon="d2l-tier1:close-large-thick" ?hidden="${!this.deletable}" @click="${this._onDeleteItem}"></d2l-icon>
```

Reply:
why can't we get rid of `aria-hidden` completely if we already have a `?hidden`? is it just because sometimes `hidden` won't be there at all, since it'll be falsy?

Reply:
I think this is the right thing to do. And, Max, yes. We basically want them to be opposites which will keep it always hidden from the screen reader.